oak_functions_containers_app/build.rs

```diff
@@ -26,6 +26,7 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
         ],
         CodegenOptions {
             build_server: true,
+            build_client: true,
```

Comment:
I guess this is just for tests? Maybe add a comment

Reply:
Yep, just for tests. Ideally this would be built only for tests, but as far as I can see the only way how to do it would be to create a separate tests crate for this.